doc/tutorials/dnn/dnn_yolo/dnn_yolo.markdown

```diff
@@ -0,0 +1,52 @@
+YOLO DNNs  {#tutorial_dnn_yolo}
+===========================
+
+Introduction
+------------
+
+In this text you will learn how to use opencv_dnn module using yolo_object_detection (Sample of using OpenCV dnn module in real time with device capture, video and image).
+
+We will demonstrate results of this example on the following picture.
+![Picture example](images/yolo.jpg)
+
+Examples
+-----------
+
+For compile and use:
+
+VIDEO DEMO:
+@youtube{NHtRlndE2cg}
+
+How to compile in comand line:
+
+@code{.bash}
+
+g++ `pkg-config --cflags opencv` `pkg-config --libs opencv` yolo_object_detection.cpp -o yolo_object_detection
+
+@endcode
+
+Execute in webcam:
+
+@code{.bash}
+
+$ yolo_object_detection -camera_device=0  -cfg=[PATH-TO-DARKNET]/cfg/yolo.cfg -model=[PATH-TO-DARKNET]/yolo.weights   -class_names=[PATH-TO-DARKNET]/data/coco.names
```

Comment:
Please use '$' symbol consistently (remove it from this line or add to other command-line samples)